data_steward/tools/create_tier.py

```diff
@@ -0,0 +1,129 @@
+# Python imports
+import argparse
+import logging
+import re
+
+# Third party imports
+
+# Project imports
+
+LOGGER = logging.getLogger(__name__)
+
+
+def validate_tier_param(tier):
+    """
+    helper function to validate the tier parameter passed is either 'controlled' or 'registered'
+
+    :param tier: tier parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if tier.lower() not in ['controlled', 'registered']:
```

Comment:
Might make this a module level list so you can reference it directly in the argument parser setup and here.  Then you only need to update it in one place.

```suggestion
    if tier.lower() not in TIER_LIST:
```
where `TIER_LIST = ['controlled', 'registered']`

Reply:
Good thinking, thank you!